### ISSUE TITLE
Logstash crashed due to typo in warning logger

### ISSUE THREAD

**Issue by sfoale** [logstash-plugins/logstash-filter-elasticsearch#38]
Using the logstash-filter-elasticsearch filter I had logstash crash with the message

```
{:timestamp=>"2016-06-23T12:15:03.473000+0000", :message=>"Exception in pipelineworker, the pipeline stopped processing new events, please check your filter configuration and restart Logstash.", "exception"=>#<ArgumentError: Cabin::Channel#warn(message, data={}) takes at most 2 arguments>, "backtrace"=>["/opt/logstash/vendor/bundle/jruby/1.9/gems/cabin-0.8.1/lib/cabin/mixins/logger.rb:62:in `warn'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-filter-elasticsearch-2.1.0/lib/logstash/filters/elasticsearch.rb:100:in `filter'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-filter-elasticsearch-2.1.0/lib/logstash/filters/elasticsearch.rb:90:in `filter'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/filters/base.rb:151:in `multi_filter'", "org/jruby/RubyArray.java:1613:in `each'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/filters/base.rb:148:in `multi_filter'", "(eval):331:in `cond_func_10'", "org/jruby/RubyArray.java:1613:in `each'", "(eval):328:in `cond_func_10'", "(eval):177:in `filter_func'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/pipeline.rb:267:in `filter_batch'", "org/jruby/RubyArray.java:1613:in `each'", "org/jruby/RubyEnumerable.java:852:in `inject'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/pipeline.rb:265:in `filter_batch'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/pipeline.rb:223:in `worker_loop'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.3.2-java/lib/logstash/pipeline.rb:201:in `start_workers'"], :level=>:error}
```

I think this is due to a typo in [elasticsearch.rb](https://github.com/logstash-plugins/logstash-filter-elasticsearch/blob/master/lib/logstash/filters/elasticsearch.rb) line 101 which reads

`      @logger.warn("Failed to query elasticsearch for previous event", :index, @index, :query => query_str, :event => event, :error =>` e)

but should be 

`      @logger.warn("Failed to query elasticsearch for previous event", :index => @index, :query => query_str, :event => event, :error => e)`

**Comment by sfoale** [comment]
I created a pull request fixing the typo.

**Comment by purbon** [comment]
Fixed in #39
